dp-terraform/helm/rhacs-terraform/values.yaml

```
@@ -202,7 +202,7 @@ external-secrets:
   installCRDs: false
   image:
     repository: quay.io/app-sre/external-secrets
-    tag: v0.9.18
+    tag: v.0.14.4
```

Does this need a dot? Typo?
```suggestion
    tag: v0.14.4
```